TheBackBenchers/Helper.cs

```diff
@@ -0,0 +1,282 @@
+                }
+                else if (node.Name == "ConnectionString")
+                {
+                    ConnectionString = node.InnerText;
+                }
+                else if (node.Name == "ContainerName")
+                {
+                    ContainerName = node.InnerText;
+                }
+                else if (node.Name == "DirectoryForListing")
+                {
+                    DirectoryForListing = node.InnerText;
+                }
+                else if (node.Name == "DirectoryForCreatingMillionBlobs")
+                {
+                    DirectoryForCreatingMillionBlobs = node.InnerText;
+                }
+                else if (node.Name == "RedirectOutputToFile")
+                {
+                    RedirectOutputToFile = bool.Parse(node.InnerText);
+                    if (RedirectOutputToFile)
+                    {
+                        Console.WriteLine("\n Output redirection was set to true - redirecting the ouput to file. \n");
+                    }
+                }
+            }
+        }
+
+        // Properties that are set by reading XML document
+        public string TenantId { get; set; }
+        public string ClientId { get; set; }
+        public string ClientSecret { get; set; }
+        public string StorageAccountName { get; set; }
+        public string BlobStorageAccountName { get; set; }
+        public string ConnectionString { get; set; }
+        public string ContainerName { get; set; }
+        public string DirectoryForListing { get; set; }
+        public string DirectoryForCreatingMillionBlobs { get; set; }
+        public bool RedirectOutputToFile { get; set; }
+
+
+        // Constants which are used at multiple places but can be fixed
+        public const int Kilobyte = 1024;
+        public const int Megabyte = 1024 * Kilobyte;
+        public const long Gigabyte = 1024 * Megabyte;
+
+        // Public variables to store results if required
+        public List<string> directoryList = new List<string> ();
+        public List<string> fileList = new List<string>();
+
+        /// <summary>
+        /// Create a container
+        /// </summary>
+        public void CreateFileSystem(DataLakeServiceClient serviceClient, string containerName)
+        {
+            // Create the filesystem and swallow already exists error
+            Console.WriteLine("Creating filesystem = {0}.", containerName);
+
+            try
+            {
+                var r = serviceClient.CreateFileSystemAsync(containerName).GetAwaiter().GetResult();
+            }
+            catch (Exception e)
+            {
+                // Do not throw on already exists exception
+                if (e.Message.Contains("ContainerAlreadyExists"))
+                {
+                    Console.WriteLine("Filesystem already exists.");
+                }
+                else
+                {
+                    throw e;
+                }
+            }
+        }

```

Comment:
Most of this logic already lives in `DataLakeFilesystemClient.CreateIfNotExistsAsync`, we catch the `ContainerAlreadyExists` Exception there.